src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -506,4 +506,7 @@
   <data name="BufferMaximumSizeExceeded" xml:space="preserve">
     <value>Cannot allocate a buffer of size {0}.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="CannotSerializeOpenGeneric" xml:space="preserve">
+    <value>Serialization or deserialization of types that contain generic parameters is invalid: '{0}'.</value>
```

Comment:
Is clarifying that {0} is a type more descriptive?

`The type {0} has generic parameters which is not valid during serialization or deserialization.`

Reply:
"Generic paremeters" sounds like any generic type is unsupported. Should we explicitly state "open generic parameters"?

Reply:
Thanks - given your feedback and [the docs](https://docs.microsoft.com/dotnet/api/system.type.containsgenericparameters?view=netcore-3.1), I went with 

```
The type '{0}' has generic parameters that have not been replaced by specific types, which is not valid during serialization or deserialization.
```